components/dashboard/table/Filter.js

```diff
@@ -0,0 +1,23 @@
+function Filter({ data }) {
```

Comment:
We already have a component for this: https://github.com/asyncapi/website/blob/master/components/form/Select.js